app/controllers/settings.js

```diff
@@ -47,6 +47,12 @@ export default Controller.extend({
       || settings.hasOwnProperty('auto_cancel_pull_requests');
   }),
 
+  showAllowConfigImportsSwitch: computed('model.settings', 'repo.private', function () {
+    let settings = this.get('model.settings');
+    let isPrivate = this.get('repo.private');
+    return isPrivate && settings.hasOwnProperty('allow_config_imports');
```

Comment:
For future reference, I would suggest to avoid relying to API data shape for UI behavior. It's out of our control and it's not covered by any tests as we don't call real API in tests. This means, it may change someday and UI will just break silently. Also it would be really hard to track and debug such errors.